web/src/components/tasks/business-structure/BusinessStructureTask.tsx

```diff
@@ -91,6 +92,10 @@ export const BusinessStructureTask = (props: Props): ReactElement => {
       })
       .queueTaskProgress({ [props.task.id]: "NOT_STARTED" });
 
+    if (business) {
```

Comment:
pebble - what is the purpose of this addition? And where is the accompanying test?

Reply:
Removed and implemented logic in `userRouter.ts` instead, because that is where the data is being reset (clearing the address fields) by way of `formationFormData: createEmptyFormationFormData()`.